src/v/redpanda/application.cc

```diff
@@ -1014,6 +1016,13 @@ void application::wire_up_redpanda_services(model::node_id node_id) {
           cloud_configs.local().bucket_name,
           std::ref(cloud_storage_api))
           .get();
+
+        construct_service(
+          _archival_upload_housekeeping, std::ref(cloud_storage_api))
```

Comment:
Should pass in the archival scheduling group?

Reply:
yes, will add this in the followup